README.adoc

```diff
@@ -159,3 +185,33 @@ At least one check failed.
 `error`::
 An error occured, for example invalid input or an unexpected state.
 [horizontal!]
+
+=== Invoking custom validators
+If the user wants to run the validator on a list of separate `.java` or `class` files, they need to be passed via the `--cp` or the `--sp` flags.
```

Comment:
In sentence "running validator on something", "something" is the thing validator inspects.